borg/archiver.py

```diff
@@ -1229,6 +1245,9 @@ def build_parser(self, args=None, prog=None):
         subparser.add_argument('--archives-only', dest='archives_only', action='store_true',
                                default=False,
                                help='only perform archives checks')
+        subparser.add_argument('--verify-data', dest='verify_data', action='store_true',
+                               default=False,
+                               help='perform full repository integrity verification (conflicts with --repository-only)')
```

Comment:
It's now "perform cryptographic archive data integrity verification" (consistent with the log messages)